style2.css

```diff
@@ -0,0 +1,315 @@
+@media screen and (max-width: 1440px) {
+	.main__pic {
+		margin: -100px 0 0 0 ;
+	}
+}
+
+@media screen and (max-width: 1120px) {
+  .header__menu-link {
+		font-size: 14px;
+	}
+  .main__pic {
+		margin: -80px 0 0 0 ;
+	}
+}
+@media screen and (max-width: 1090px) {
+  .header__menu-link {
+	font-size: 13px;
+}
+  .main__pic {
+		margin: -70px 0 0 0 ;
+	}
+}
+
+@media screen  and (max-width: 1062px) {
+  .menu{
+	display: none;
+}
+  .header__menu-link {
+	display: none;
+}
+}
+
+
+
+@media screen and (max-width: 1024px) {
+  .body {
+text-align: center;
+align-items: center;
+}
+
+  .main__pic {
+		margin: -60px 0 0 0 ;
+	}
+	
+  .header {
+diplay: flex;
+padding: 0;
+margin: 0 auto;
+}
+
+  .main {
+padding: 0;
+margin: 0 auto;	
+}
+
+
+  .header__menu {
+display: none;
+}
+
+  .header__menu-link {
+	display: none;
+
+}
+
+  .header__line {
+	display: none;
+}
+
+  .main__title {
+	font-size: 90px;
+	margin-bottom: 10px;	
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/tesla394/pullRequest?prid=5642156)